common/main/be.xml

```diff
@@ -10147,126 +10147,126 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>Гц</displayName>
+				<unitPattern count="one">{0} Гц</unitPattern>
+				<unitPattern count="few">{0} Гц</unitPattern>
+				<unitPattern count="many">{0} Гц</unitPattern>
+				<unitPattern count="other">{0} Гц</unitPattern>
 			</unit>
 			<unit type="graphics-em">
-				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>эм</displayName>
+				<unitPattern count="one">{0} эм</unitPattern>
+				<unitPattern count="few">{0} эм</unitPattern>
+				<unitPattern count="many">{0} эм</unitPattern>
+				<unitPattern count="other">{0} эм</unitPattern>
 			</unit>
 			<unit type="graphics-pixel">
-				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>пікселі</displayName>
+				<unitPattern count="one">{0} піксель</unitPattern>
+				<unitPattern count="few">{0} пікселі</unitPattern>
+				<unitPattern count="many">{0} пікселяў</unitPattern>
+				<unitPattern count="other">{0} пікселя</unitPattern>
 			</unit>
 			<unit type="graphics-megapixel">
-				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>Мпікс</displayName>
+				<unitPattern count="one">{0} Мпікс</unitPattern>
+				<unitPattern count="few">{0} Мпікс</unitPattern>
+				<unitPattern count="many">{0} Мпікс</unitPattern>
+				<unitPattern count="other">{0} Мпікс</unitPattern>
 			</unit>
 			<unit type="graphics-pixel-per-centimeter">
-				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>піксель/см</displayName>
+				<unitPattern count="one">{0} піксель/см</unitPattern>
+				<unitPattern count="few">{0} пікселі/см</unitPattern>
+				<unitPattern count="many">{0} пікселяў/см</unitPattern>
+				<unitPattern count="other">{0} пікселя/см</unitPattern>
 			</unit>
 			<unit type="graphics-pixel-per-inch">
-				<displayName>↑↑↑</displayName>
-				<unitPattern count="one">↑↑↑</unitPattern>
-				<unitPattern count="few">↑↑↑</unitPattern>
-				<unitPattern count="many">↑↑↑</unitPattern>
-				<unitPattern count="other">↑↑↑</unitPattern>
+				<displayName>піксель/цалю</displayName>
+				<unitPattern count="one">{0} піксель/цалю</unitPattern>
+				<unitPattern count="few">{0} пікселі/цалю</unitPattern>
```

Comment:
"modify -fF" reports:

Resolved values differ: vxml [{0} пікселі/цалю] differs from baseline [{0} піксэлі/цалю] for be_TARASK //ldml/units/unitLength[@type="narrow"]/unit[@type="graphics-pixel-per-inch"]/unitPattern[@count="few"]

Note, the difference is "е" versus "э"... replacement of soft with hard value here evidently changes what gets inherited by be_TARASK

```
$ egrep "піксэлі/цалю" common/main/be_TARASK.xml
				<unitPattern count="few" draft="provisional">{0} піксэлі/цалю</unitPattern>
```

Reply:
if it's already hard-coded in be_TARASK.xml, how can inheritance make any difference?

Reply:
likely answer to my own question above, is that it's already hard-coded for at least one path but other paths in the same child locale inherit from it laterally; but if ^^^ changes to hard value in the parent then the child inherits vertically instead of laterally